api/src/pcapi/core/booking_providers/cds/client.py

```diff
@@ -15,7 +15,7 @@
 from pcapi.core.booking_providers.cds.constants import VOUCHER_PAYMENT_TYPE_CDS
 import pcapi.core.booking_providers.cds.exceptions as cds_exceptions
 from pcapi.core.booking_providers.models import BookingProviderClientAPI
-from pcapi.core.booking_providers.models import SeatCDS
+from pcapi.core.booking_providers.models import Movie
 from pcapi.core.booking_providers.models import SeatMap
 from pcapi.core.booking_providers.models import Ticket
```

Comment:
```suggestion
import pcapi.core.booking_providers.models as booking_providers_models
```
cf commentaire précédent